### PR TITLE
WIP: Integrated Antenatal Register Validation Rules

### PR DESCRIPTION
**Pull request by SokoPaulSokool**
### What does this PR do
- adds validation rules to the Integrated Antenatal Register

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml [hunk 1]
```diff
@@ -1,4 +1,51 @@
 <htmlform formName="HMIS 005 Integrated Antenatal Register" formDescription="Entry form for patients enrolled for ANC services" formEncounterType="044daI6d-f80e-48fe-aba9-037f241905Pe" formUuid="362e296b-1315-48a9-8de6-5f4cc3cad314" formVersion="3.0.0" formAddMetadata="yes" formUILocation="patientDashboard.visitActions" formOrder="8" formIcon="icon-medkit" formShowIf="(visit.active || !visit.active) &amp;&amp; patient.person.gender=='F' &amp;&amp; patient.person.age &gt; 9 &amp;&amp; patient.person.dead==false &amp;&amp; sessionLocation.uuid=='e9bc61b5-69ff-414b-9cf0-0c22d6dfca2b'" formDisplayStyle="Standard" formLabel="ANC Encounter Form">
+    <script type="text/javascript">
+    //Disable encounter date field.
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml]: @SokoPaulSokool make this a global function in the JS file work with @semujju as its used on almost every page

File: omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml [hunk 2]
```diff
@@ -1,4 +1,51 @@
...
+
+                jq('#viral-load-results').find('select').removeAttr("disabled");
+                jq('#viral-load-results').find('select').removeClass('disabled');
+                
+                jq('#viral-load-status').find('select').removeAttr("disabled");
+                jq('#viral-load-status').find('select').removeClass('disabled');
+            }
+            disableViralLoadFields();
+            jq('#163023').find('#w44').change(function($event){
+                console.log($event.target.value);
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml]: @SokoPaulSokool remove the console.log

File: omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml [hunk 3]
```diff
@@ -1,4 +1,51 @@
...
+                jq('#viral-load').find('input[type="text"]').removeClass('disabled');
+
+                jq('#viral-load-results').find('select').removeAttr("disabled");
+                jq('#viral-load-results').find('select').removeClass('disabled');
+                
+                jq('#viral-load-status').find('select').removeAttr("disabled");
+                jq('#viral-load-status').find('select').removeClass('disabled');
+            }
+            disableViralLoadFields();
+            jq('#163023').find('#w44').change(function($event){
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml]: @SokoPaulSokool you can attach a JS file by using the functionality at https://wiki.openmrs.org/display/docs/HTML+Form+Entry+JavaScript+Reference#HTMLFormEntryJavaScriptReference-getField(idAndProperty) 

@solemabrothers

File: omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml (hunk 1 above)
Comment (SokoPaulSokool) [reply omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml]: done

File: omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml (hunk 2 above)
Comment (SokoPaulSokool) [reply omod/src/main/webapp/resources/htmlforms/HMIS-005-IntegratedAntenatalRegister.xml]: done

**Pull request merged**